bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -335,7 +335,8 @@ private boolean validateOptionValues(Map<String, ConfigurationOption> options, M
 
     private void setDefaultOutputPath(Map<String, Object> optionValues)
     {
-        if (!optionValues.containsKey(OutputPathOption.NAME))
+        Object obj = optionValues.getOrDefault(OutputPathOption.NAME, null);
+        if (obj == null || (obj instanceof File && StringUtils.isBlank( ((File) obj).getPath())) )
         {
```

Comment:
If you launch:
`rhamt-cli --input something --output     --target eap`
this should avoid it and let set the default output directory

Reply:
ok